cli/azd/pkg/tools/azcli/function_app.go

```diff
@@ -44,6 +49,37 @@ func (cli *azCli) DeployFunctionAppUsingZipFile(
 		return nil, err
 	}
 
+	cred, err := cli.credentialProvider.CredentialForSubscription(ctx, subscriptionId)
+	if err != nil {
+		return nil, err
+	}
+
+	plansClient, err := armappservice.NewPlansClient(subscriptionId, cred, cli.armClientOptions)
+	if err != nil {
+		return nil, err
+	}
+
+	planId := *app.Properties.ServerFarmID
+	sep := strings.LastIndexByte(planId, '/')
+	planName := planId[sep+1:]
```

Comment:
should we check sep != -1 as an error  here?

Reply:
Makes sense